packages/plugins/experimental/plugin-canvas/src/components/CanvasContainer.tsx

```diff
@@ -7,20 +7,20 @@ import React from 'react';
 import { fullyQualifiedId } from '@dxos/react-client/echo';
 import { StackItem } from '@dxos/react-ui-stack';
 
-import { AttentionContainer } from './AttentionContainer';
 import { Editor } from './Editor';
+import { KeyboardContainer } from './KeyboardContainer';
 import { type CanvasBoardType } from '../types';
 
-export const CanvasContainer = ({ canvas }: { canvas: CanvasBoardType }) => {
+export const CanvasContainer = ({ canvas, role }: { canvas: CanvasBoardType; role: string }) => {
   const id = fullyQualifiedId(canvas);
   return (
-    <StackItem.Content id={id} toolbar={false}>
-      <AttentionContainer id={id}>
```

Comment:
Do I understand correctly that stack/deck plugins wrap the surface in an attendable div, so while the AttentionContainer might be necessary for testing styling in stories, it's not needed in the plugin?

Reply:
Yes exactly, DeckPlugin and StackPlugin handle attention for articles and sections respectively.